www/src/utils/node/docs.js

```diff
@@ -193,15 +189,6 @@ exports.createPages = async ({ graphql, actions }) => {
             ...prevAndNext,
           },
         })
-      } else if (node.fields.package) {
```

Comment:
This never actually gets called because the NPM packages don't go through the MDX transformer!